imagetest/test_suites/shapevalidation/setup.go

```diff
@@ -31,6 +34,7 @@ var x86shapes = map[string]*shape{
 		numa:  4,
 		disks: []*compute.Disk{{Name: "C3", Type: imagetest.PdBalanced, Zone: "us-east1-b"}},
 		zone:  "us-east1-b",
+		quota: []*daisy.QuotaAvailable{&daisy.QuotaAvailable{Units: 176.0, Metric: "C3_CPUS", Region: "us-east1"}},
```

Comment:
Do we want to use a hard coded region?

Reply:
I don't know if we currently have flags for it though.

Reply:
We are already hard-coding the region, this is just adding the quota code. Hard-coding the region is not a perfect solution, but it's necessary for now to be able to run tests on VMs that are only available in certain zones. There is no region with enough quota to support every machine type we need for these suites so they will have to diverge somehow.

A better solution would be the ability to auto-determine region based on the requirements of the workflow but this would required a pretty big refactor to incorporate the compute API more deeply into the workflow. We mostly parse strings for information right now.

VMs without hard coded regions still respect the flag passed on the command line.